builtin/plugins/dposv3/dpos.go

```diff
@@ -1336,6 +1337,27 @@ func (c *DPOS) ListAllDelegations(ctx contract.StaticContext, req *ListAllDelega
 	}, nil
 }
 
+func (c *DPOS) EnableValidatorJailing(ctx contract.Context, req *EnableValidatorJailingRequest) error {
+	sender := ctx.Message().Sender
+	if !ctx.FeatureEnabled(loomchain.DPOSVersion3_4, false) {
+		return errors.New("DPOS v3.4 is not enabled")
+	}
+
+	state, err := LoadState(ctx)
+	if err != nil {
+		return err
+	}
+	if state.Params.OracleAddress == nil || sender.Compare(loom.UnmarshalAddressPB(state.Params.OracleAddress)) != 0 {
+		return errors.New("Invalid Oracle Address")
```

Comment:
Looks like the tests broke:
```
--- FAIL: TestChangeParams (0.00s)
[2019-07-12T08:37:30.367Z]     require.go:157: 
[2019-07-12T08:37:30.367Z]         	Error Trace:	dpos_test.go:189
[2019-07-12T08:37:30.367Z]         	Error:      	Not equal: 
[2019-07-12T08:37:30.367Z]         	            	expected: &errors.errorString{s:"Function can only be called with oracle address."}
[2019-07-12T08:37:30.367Z]         	            	actual  : &errors.errorString{s:"Invalid Oracle Address"}
[2019-07-12T08:37:30.367Z]         	            	
[2019-07-12T08:37:30.367Z]         	            	Diff:
[2019-07-12T08:37:30.367Z]         	            	--- Expected
[2019-07-12T08:37:30.367Z]         	            	+++ Actual
[2019-07-12T08:37:30.367Z]         	            	@@ -1,2 +1,2 @@
[2019-07-12T08:37:30.367Z]         	            	-(*errors.errorString)(Function can only be called with oracle address.)
[2019-07-12T08:37:30.367Z]         	            	+(*errors.errorString)(Invalid Oracle Address)
[2019-07-12T08:37:30.367Z]         	            	 
[2019-07-12T08:37:30.367Z]         	Test:       	TestChangeParams
```

Probably should just `return errOnlyOracle` instead here.